content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
+These platforms typically offer version control, multiple permission levels, and other functionality to support multi-author document editing.
+Although they work well for editing, they lack sufficient features for managing a collaborative manuscript and attributing precise credit, which are important for open writing.
+
+We adopted standard software development strategies in order to enable any contributor to edit any part of the manuscript but enforce discussion and review of all proposed changes.
+The GitHub platform provided support for organizing and editing the manuscript.
+We used GitHub _issues_ for organization, opening a new issue for each paper under consideration.
+Within the issue, contributors summarized the research, discussed it (sometimes with the original authors), and assessed its relevance to the review.
+Issues also served as an open to-do list and a forum for debating the main message, themes, and topics of the review.
+
+GitHub and the underlying git version control system also structured the writing process.
+An official master version is _forked_ by individual contributors.
+A contributor then adds and revises files, grouping these changes into _commits_.
+When the changes are ready to be reviewed, the series of commits are submitted as a _pull request_ through GitHub, which notifies other authors of the pending changes.
+GitHub's review interface allows anyone to comment on the changes, globally or at specific lines, asking questions or requesting modifications as depicted in [@url:https://medium.com/towards-data-science/opportunities-and-obstacles-for-deep-learning-in-biology-and-medicine-6ec914fe18c2].
+Conversations during review can reference other pull requests, issues, or authors, linking the relevant people and content.
+Reviewing batches of revisions that focus on a single theme is more efficient than independently discussing isolated comments and edits, and it helps maintain consistent content and tone across different authors and reviewers.
+Once all requested modifications are made, the manuscript maintainers, a subset of authors with elevated GitHub permissions, formally approve the pull request and merge the changes into the official master version.
+`TODO: need a figure with a flowchart showing this process`
+The entire process can be orchestrated through GitHub with a web browser if a contributor does not want to use a git client on their own computer.
+
+We found that this workflow was an effective compromise between fully unrestricted editing and a more heavily-structured approach that limits the authors or the sections they can edit.
+In addition, authors are associated with their commits, which makes it easy for contributors to receive credit for their work and prevents ghostwriting [@doi:10.1371/journal.pmed.1000023].
+The GitHub contributors page summarizes all edits and commits from each author, providing aggregated information that is not available on other collaborative writing platforms.
```

Comment:
I'll work on a revision to note this.  I also want to be fair to other platforms that also have git/gitHub integration and phrase things accordingly.

Authorea [syncs to GitHub](https://intercom.help/authorea/user-settings/user-settings/syncing-articles-to-github) ([example](https://github.com/jdblischak/git-for-science)) and Overleaf has some [support for GitHub](https://www.overleaf.com/blog/195-new-collaborate-online-and-offline-with-overleaf-and-git-beta#.WYnKg4jythE) that I haven't looked at carefully.